data/json/monsters.json

```diff
@@ -2340,7 +2340,43 @@
     "vision_night": 3,
     "harvest": "mr_bones",
     "special_attacks": [ [ "scratch", 10 ], { "type": "bite", "cooldown": 5 } ],
-    "upgrades": { "half_life": 15, "into": "mon_skeleton_hulk" },
+    "upgrades": { "half_life": 15, "into": "mon_skeleton_brute" },
+    "death_drops": "default_zombie_clothes",
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "BLEED", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
+  },
+  {
+	"//": "Skely brute gaps the evolution between normal and the juggernaut.",
+    "id": "mon_skeleton_brute",
+    "type": "MONSTER",
+    "name": "skeletal brute",
+    "description": "Distorted outgrowths of calcified bone plates covers this zombie's rotten skin. Its joints and cracks around its body oozes with black goo.",
```

Comment:
Distorted outgrowths of calcified bone plates cover this zombie's rotten skin. Joints and cracks around its body ooze with black goo.

Reply:
Done!